docs/src/components/HireUsSection/styles.module.css

```diff
@@ -0,0 +1,71 @@
+.hireUsSectionWrapper {
+  display: flex;
+  position: relative;
+  flex-direction: column;
+  align-items: center;
+  gap: 2.5rem;
+  width: 60%;
+  margin: 16rem auto 12rem auto;
+  padding-bottom: 2rem;
```

Comment:
```suggestion
  padding-bottom: 2rem;
  text-wrap: balance;
```
🙏 

![image](https://github.com/software-mansion/react-native-reanimated/assets/39658211/1e8636a4-5a7d-46f3-99e6-0ada1db4ce6a)